tests/4.5/task/test_task_target.c

```diff
@@ -0,0 +1,58 @@
+//===--- test_target_map_pointer.c - test target with map pointer p[:N] -----===//
+// 
+// OpenMP API Version 4.5 Nov 2015
+//
+// This test checks target regions inside of tasks. With offloading support 
+// the tasks are executed on the device. The results do not change if the 
+// test is executed on the host. Output will indicate if the test passed and
+// where (host/device).
+//
+////===----------------------------------------------------------------------===//
+
+#include <stdio.h>
+#include <omp.h>
+#include "ompvv.h"
+
+#define N 1000
+
+int test_task_target(){
```

Comment:
Minor coding standard fix: space between close paren and open curly brace (also on lines 44 and 50)

Reply:
Addressed.